CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Change log
 -----------
 
+# v2.0.4+rev2 - 2017-05-28
```

Comment:
Why do we have two releases in this PR? We could include the uboot change in rev1 too.